test-utils/src/main/java/com/criteo/publisher/network/CdbMock.kt

```diff
@@ -347,12 +349,73 @@ class CdbMock(private val jsonSerializer: JsonSerializer) {
     """.trimIndent()
 
       return MockResponse()
-          .setHeader(CONTENT_TYPE, "text/plain; charset=utf-8")
+          .setHeader(CONTENT_TYPE, TEXT_PLAIN)
+          .setBody(response)
+    }
+
+    @Suppress("MaxLineLength")
+    private fun handleVastCasperRequest(request: RecordedRequest): MockResponse {
+      val response = """
+        <?xml version="1.0" encoding="utf-8"?>
+        <VAST version="3.0">
+          <Ad>
+            <Wrapper>
```

Comment:
Response in current `handleVastCasperRequest`

Reply:
What do you mean ? I didn't get your point, sorry 😕

Reply:
You can ignore this comment, I just wanted to include the code snippet.